recipes/orca/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "orca" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "c997eef9ce4ad8a2ae6f7c5b9fdc62db29d29f43dc75c77826e13a624ceac4c9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/UDST/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - orca-server = orca.server.server:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
```

Comment:
I don't seen any c-extensions being built , so I'm not sure that you need `toolchain`. Also, it looks like all you need is `python` and `setuptools` since the others aren't actually build time requirements.